src/cosp.F90

```diff
@@ -384,7 +385,7 @@ function COSP_SIMULATOR(cospIN,cospgridIN,cospOUT,start_idx,stop_idx,debug)
     real(wp),dimension(:,:,:),allocatable :: &
        betamol_in,betamoli,pnormi,ze_toti,ze_noni
     real(wp),dimension(:,:,:),allocatable :: &
-         t_in,tmpFlip,ze_totFlip
+         t_in,tempI,frac_outI
```

Comment:
Can these variable names be made more informative? Can you add comments explaining what they are?

Reply:
These variable names are following to the CloudSat/CALIPSO joint-product. I added a comment line explaining these variables.